.yamllint

```diff
@@ -0,0 +1,7 @@
+---
+extends: default
+ignore: .travis.yml
```

Comment:
We probably don't need this bit

Reply:
If we're using yamlint, it always seems to find those files.

Reply:
there shouldn't be a travis config in this repo though